BitwardenShared/UI/Platform/Settings/Settings/AccountSecurity/AccountSecurityProcessor.swift

```diff
@@ -166,6 +168,9 @@ final class AccountSecurityProcessor: StateProcessor<
             if try await services.authRepository.isPinUnlockAvailable() {
                 state.isUnlockWithPINCodeOn = true
             }
+            state.isAuthenticatorSyncEnabled = try await services.stateService.getSyncToAuthenticator()
+            state.shouldShowAuthenticatorSyncSection =
+                await services.configService.getFeatureFlag(.enableAuthenticatorSync)
```

Comment:
🎨 Even though it may be a little, we can improve the performance of loading this if the feature flag is off then we don't need to call `try await services.stateService.getSyncToAuthenticator()`; so:
```suggestion
            state.shouldShowAuthenticatorSyncSection =
                await services.configService.getFeatureFlag(.enableAuthenticatorSync)
            if state.shouldShowAuthenticatorSyncSection {
                state.isAuthenticatorSyncEnabled = try await services.stateService.getSyncToAuthenticator()
            }
```

Reply:
👍  Good optimization. Added that in the latest commit.